cdc/tests/integration_tests/flow_control/run.sh

```diff
@@ -19,9 +20,10 @@ function run() {
 
 	start_ts=$(tikv-cdc cli tso query --pd=$UP_PD)
 	sleep 10
+	go-ycsb load tikv -P $CUR/config/workload -p tikv.pd="$UP_PD" -p tikv.type="raw" -p tikv.apiversion=V2 -p recordcount=1000000 -p operationcount=1000000 --threads 100 # About 1G
```

Comment:
Nit: the `recordcount` can be written in `workload` file. As we use an individual workload file, it's not necessary to specify this property here. 
And `operationcount` is useless in `go-ycsb load`.

Reply:
Thanks, I removed `operationcount` and moved `receordcount` to `workload`.